packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -614,7 +629,7 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
     private updateGCNodes(builder: GCDataBuilder) {
         // Add a back route to self in each child's GC nodes. If any child is referenced, then its parent should
         // be considered referenced as well.
-        builder.addRouteToAllNodes(this.absolutePath);
```

Comment:
Following up on the above comment, the route added here should be - `/_channels/<datastoreId>`